src/Components/Publishing/Header/AuthorDateClassic.tsx

```diff
@@ -33,14 +33,18 @@ const getAuthorByline = (authors, author) => {
     } else {
       formattedAuthor = authors[0].name
     }
+
     return (
       <div>
-        <TextSm>{author.name}</TextSm>
+        {/* FIXME: <TextSm>{author.name}</TextSm> */}
+        <TextSm>FIXME</TextSm>
         <div>{"By " + formattedAuthor}</div>
       </div>
     )
   } else {
-    return author.name
+    // FIXME
+    // return author.name
+    return "FIXME"
```

Comment:
@kanaabe - it looks like some classic articles break on this data and i'm not too sure....

Reply:
Positron only uses this component rn -- did you check by passing in `contributing_authors`? Not sure why this would break

Reply:
Ahh, undid -- just going to leave alone for now as I'm not sure what was up.